app/views/shared/means/_student_finances.html.erb

```diff
@@ -4,7 +4,8 @@
 <%= page_template page_title: page_title, form: form, template: :basic do %>
   <%= form.govuk_radio_buttons_fieldset :student_finance,
                                         hint: {text: t('.hint')},
-                                        legend: {text: content_for(:page_title), tag: 'h1', size: 'xl'} do %>
+                                        legend: {text: content_for(:page_title), tag: 'h1',
```

Comment:
Is `size` on its own line for a reason?
If so, shouldn't it be 
```suggestion
                                        legend: {text: content_for(:page_title), 
                                        tag: 'h1',
                                        size: 'xl'} do %>
```
and all aligned?  Size seems to be an extra space in!